mtp_api/apps/mtp_auth/models.py

```diff
@@ -21,7 +32,16 @@ class ApplicationUserMapping(TimeStampedModel):
     application = models.ForeignKey('oauth2_provider.Application')
 
     def __str__(self):
-        return self.user.username
+        return '%s -> %s' % (self.user.username, self.application.client_id)
+
+
+class ApplicationGroupMapping(TimeStampedModel):
```

Comment:
i didn't on the basis that there is no fixture for the default set of applications